cmd/okctl/completion.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"github.com/oslokommune/okctl/pkg/okctl"
+	"github.com/spf13/cobra"
+)
+
```

Comment:
Need to add a `// nolint: funlen` to this function, I suggest running `make check` locally